nav2_behavior_tree/test/plugins/control/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+include_directories(../../)
```

Comment:
Why not in the root test cmakelists file before the others here are called so they're included?